util.go

```diff
@@ -31,3 +31,6 @@ func CertName(cert *x509.Certificate) string {
 	sum := sha256.Sum256(cert.Raw)
 	return base64.RawURLEncoding.EncodeToString(sum[:])
 }
+
+// ValidityMapURL
```

Comment:
I take it this is required by the linter? (No action required, just venting.)

Reply:
Updated since we care about the trailing slash in packagebase